### PR TITLE
Adds a who command

### PR DESCRIPTION
**Pull request by orta**
Adds docs on how you can make a slack token, took longer than making the PR for me.

Adds a who command that does a comma separated list of the users on the db.

### CODE REVIEW

File: docs/slack_token.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Getting a token
+
+![](./token.png)
```
Comment (orta) [inline docs/slack_token.md:3]: the token in the image isn't usable

**Comment by alloy** [comment]
👏

**Comment by orta** [comment]
hah

doh

screen shot 2017-01-11 at 11 23 46 am

**Comment by alloy** [comment]
Oh, thought I had reported that... my bad

**Pull request merged**